CHANGELOG.md

```diff
@@ -28,7 +29,7 @@ Breaking changes:
 
 New features:
 - Added `nubEq`/`nubByEq` (#179)
-- Added `groupAllBy` (#182, #191) 
```

Comment:
I prematurely added this PR to the changelog in #195. I was hoping this PR would have been included in the batch of breaking changes, so the release history would see `groupAllBy` as being added correctly the first time.